server/src/main/java/com/pepperkick/ems/server/config/IgniteCacheConfiguration.java

```diff
@@ -0,0 +1,60 @@
+package com.pepperkick.ems.server.config;
+
+import com.pepperkick.ems.server.entity.Employee;
+import io.github.cdimascio.dotenv.Dotenv;
```

Comment:
this is kind of alternative to Dotenv : https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/core/env/Environment.html